gcpy/budget_tt.py

```diff
@@ -559,17 +559,17 @@ def annual_average_sources(globvars):
         raise ValueException(msg)
 
     # Convert Be7 and Be10 sources from kg/m2/s to g/day
-    # NOTE: This is a kludgey way to do it but it works and
-    # preserves the shape of the data as (time,lev,lat,lon).
+    # If GCHP data, must vertically flip the emissions diagnostic
     for t in range(globvars.N_MONTHS):
         for k in range(n_levs):
             if globvars.is_gchp:
+                kf = k + 1 - n_levs
```

Comment:
Wondering if we should use kf = nlevs - k -1, since Python starts indices from 0
```console
72 - 0 - 1 = 71
72  - 1  -1 = 70
72  - 2 - 1 = 69
...
72 - 71 - 1 = 0
```

Reply:
kf = k + 1 - nlevs doesn't seem to give the proper flipped index
```console
>>> n_levs=72
>>> a = list(range(n_levs))
>>> print(a)
[0, 1, 2, 3. .. 71]
>>> kf = 0 + 1 - n_levs
>>> print(kf)
-71
>>> print(a[kf])
1
```
When k=0 you should get kf=71

Reply:
Whoops! I'll push a fix.